PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -256,7 +258,10 @@
     )
 for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016:
   modifier.toModify( jetTable.variables, jetId = Var("userInt('tightIdLepVeto')*4+userInt('tightId')*2+userInt('looseId')",int,doc="Jet ID flags bit1 is loose, bit2 is tight, bit3 is tightLepVeto"))
+run2_nanoAOD_102Xv1.toModify( jetTable.variables, puIdDisc = Var("userFloat('puId102XDisc')",float,doc="Pileup ID discriminant with 102X (2018) training",precision=10) )
+run2_nanoAOD_102Xv1.toModify( jetTable.variables, puId = Var("userInt('pileupJetId:fullId')",int,doc="Pileup ID flags for pre-UL trainings") )
```

Comment:
you can change line 261 and 262 into 

( (run2_nanoAOD_106Xv1 and ~run2_nanoAOD_devel) | run2_nanoAOD_102Xv1).toModify( jetTable.variables, puIdDisc = Var("userFloat('puId102XDisc')",float,doc="Pileup ID discriminant with 102X (2018) training",precision=10) )
( (run2_nanoAOD_106Xv1 and ~run2_nanoAOD_devel) | run2_nanoAOD_102Xv1).toModify( jetTable.variables, puId = Var("userInt('pileupJetId:fullId')",int,doc="Pileup ID flags for pre-UL trainings") )

Reply:
ok thanks, I'll commit a new version

Reply:
@alefisico 
can you replace `and` with `&` ?
so that we have 
( (run2_nanoAOD_106Xv1 & ~run2_nanoAOD_devel) | run2_nanoAOD_102Xv1 ).toModify(